crates/chia-sdk-driver/src/layers/bulletin_layer.rs

```diff
@@ -0,0 +1,90 @@
+use chia_sdk_types::{
+    puzzles::{IndexWrapperArgs, INDEX_WRAPPER_HASH},
+    Mod,
+};
+use clvm_traits::{FromClvm, MatchByte};
+use clvm_utils::{ToTreeHash, TreeHash};
+use clvmr::{Allocator, NodePtr};
+
+use crate::{DriverError, Layer, Puzzle, SpendContext};
+
+pub type BulletinNonce = (MatchByte<98>, ());
+
+/// The Bulletin [`Layer`] is used to wrap a puzzle hash with a static nonce
+/// to differentiate it from normal XCH coins.
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub struct BulletinLayer<I> {
+    /// The inner puzzle layer, used to identify and spend the bulletin.
+    pub inner_puzzle: I,
+}
+
+impl<I> BulletinLayer<I> {
+    pub fn new(inner_puzzle: I) -> Self {
+        Self { inner_puzzle }
+    }
+}
+
+impl<I> Layer for BulletinLayer<I>
+where
+    I: Layer,
+{
+    type Solution = I::Solution;
+
+    fn parse_puzzle(allocator: &Allocator, puzzle: Puzzle) -> Result<Option<Self>, DriverError> {
```

Comment:
I think this is saying "We can identify a bulletin as a coin that has a nonce of byte value 98 and an inner puzzle"?